selfdrive/car/chrysler/carstate.py

```diff
@@ -48,11 +48,12 @@ def update(self, cp, cp_cam):
     ret.steeringRateDeg = cp.vl["STEERING"]["STEERING_RATE"]
     ret.gearShifter = self.parse_gear_shifter(self.shifter_values.get(cp.vl["GEAR"]["PRNDL"], None))
 
-    ret.cruiseState.enabled = cp.vl["ACC_2"]["ACC_STATUS_2"] == 7  # ACC is green.
-    ret.cruiseState.available = ret.cruiseState.enabled  # FIXME: for now same as enabled
+    ret.cruiseState.enabled = cp.vl["ACC_2"]["ACC_STATUS_2"] == 7  # ACC is green
+    ret.cruiseState.available = cp.vl["ACC_2"]["ACC_STATUS_2"] != 1  # ACC is white
```

Comment:
Will move the available signal change to another PR once I get this tested by a user